File: RSPAClient/Core/AtPar.POCOEntities/MT_CYCT_EVENT_HDR_MASTER.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_CYCT_EVENT_HDR_MASTER
    {
        public string TRANSACTION_ID { get; set; }
        public string BUSINESS_UNIT { get; set; }
        public string EVENT_ID { get; set; }
        public string PARENT_EVENT_ID { get; set; }
        public Nullable<double> NO_OF_ITEMS { get; set; }
        public string UPDATE_USER_ID { get; set; }
        public Nullable<System.DateTime> UPDATE_DATE { get; set; }
        public Nullable<int> SORT_BY_FIELD { get; set; }
        public string FROM { get; set; }
        public string TO { get; set; }

        //Ignore
        public int NO_OF_SUBEVENTS { get; set; }
        public string COUNT_HDR_STATUS { get; set; }
        public int ISSPLITTED { get; set; }
        public string USER_ID { get; set; }
    }
}

File: RSPAClient/Atomic/app/Entities/MT_CYCT_EVENT_HDR_MASTER.ts
export class MT_CYCT_EVENT_HDR_MASTER {
    public TRANSACTION_ID: string;
    public BUSINESS_UNIT: string;
    public EVENT_ID: string;
    public PARENT_EVENT_ID: string;
    public NO_OF_ITEMS?: number;
    public UPDATE_USER_ID: string;
    public UPDATE_DATE?: Date;
    public SORT_BY_FIELD?: number;
    public FROM: string;
    public TO: string;
}